### PR TITLE
Fix/update drude snapshots

### PR DESCRIPTION
**Pull request by dpanta94**
### 🎫 Ticket

Fixing test snapshots

### CODE REVIEW

File: tests/ft_integration/TEC/Tickets/Flexible_Tickets/Series_Passes/__snapshots__/Emails_Test__should_correctly_render_the_email_settings_tab__0.snapshot.html [hunk 1]
```diff
@@ -84,9 +84,9 @@
...
+
 <div id="wp-tec-tickets-emails-footer-content-editor-container" class="wp-editor-container"><textarea class="wp-editor-area" style="height: 200px" cols="40" name="tec-tickets-emails-footer-content" id="tec-tickets-emails-footer-content"></textarea></div>
 </div>
 
-<label class="screen-reader-text">Add custom links and instructions to the bottom of your emails.</label><p class="tooltip description">Add custom links and instructions to the bottom of your emails.</p></div></fieldset><div class="clear"></div>
```
Comment (dpanta94) [inline tests/ft_integration/TEC/Tickets/Flexible_Tickets/Series_Passes/__snapshots__/Emails_Test__should_correctly_render_the_email_settings_tab__0.snapshot.html:92]: pointiong out this `clear` getting removed

Comment (Camwyn) [reply tests/ft_integration/TEC/Tickets/Flexible_Tickets/Series_Passes/__snapshots__/Emails_Test__should_correctly_render_the_email_settings_tab__0.snapshot.html:92]: Yes, we removed several of them - we don't use floats any more

File: tests/ft_integration/TEC/Tickets/Flexible_Tickets/Series_Passes/__snapshots__/Emails_Test__should_correctly_render_the_email_settings_tab__0.snapshot.html [hunk 2]
```diff
@@ -84,9 +84,9 @@
...
-<link rel='stylesheet' id='editor-buttons-css' href='http://wordpress.test/wp-includes/css/editor.min.css?ver=6.4.1' media='all' />
```
[the remark this answers is not in the document]
Comment (Camwyn) [reply tests/ft_integration/TEC/Tickets/Flexible_Tickets/Series_Passes/__snapshots__/Emails_Test__should_correctly_render_the_email_settings_tab__0.snapshot.html:88]: hm

**Review by Camwyn — APPROVED** [review]
Pending @codingmusician approval

File: tests/ft_integration/TEC/Tickets/Flexible_Tickets/Series_Passes/__snapshots__/Emails_Test__should_correctly_render_the_email_settings_tab__0.snapshot.html [hunk 3]
```diff
@@ -84,9 +84,9 @@
...
..."clear"></div><fieldset id="tribe-field-tec-tickets-emails-header-bg-color" class="tribe-field tribe-field-color tribe-size-medium"><legend class="tribe-field-label" >Header/Footer Background</legend><div class="tribe-field-wrap"><input type="text" class="tec-admin__settings-color-field-input" name="tec-tickets-emails-header-bg-color" value="#50B078"/></div></fieldset><div class="clear"></div><fieldset id="tribe-field-tec-tickets-emails-ticket-bg-color" class="tribe-field tribe-field-color tribe-size-medium"><legend class="tribe-field-label" >Ticket Color</legend><div class="tribe-field-wrap"><input type="text" class="tec-admin__settings-color-field-input" name="tec-tickets-emails-ticket-bg-color" value="#2D2E33"/></div></fieldset><div class="clear"></div><fieldset id="tribe-field-tec-tickets-emails-footer-content" class="tribe-field tribe-field-wysiwyg tribe-size-medium"><legend class="tribe-field-label" >Footer Content</legend><div class="tribe-field-wrap"><div id="wp-tec-tickets-emails-footer-content-wrap" class="wp-core-ui wp-editor-wrap html-active"><link rel='stylesheet' id='dashicons-css' href='http://wordpress.test/wp-includes/css/dashicons.min.css?ver=6.4.1' media='all' />
```
Comment (codingmusician) [inline tests/ft_integration/TEC/Tickets/Flexible_Tickets/Series_Passes/__snapshots__/Emails_Test__should_correctly_render_the_email_settings_tab__0.snapshot.html:87]: I'm pretty sure this shouldn't be removed. These are all the settings.

**Review by codingmusician — CHANGES_REQUESTED** [review]

**Review by codingmusician — APPROVED** [review]

**Pull request merged**